### ISSUE TITLE
Feature request: Disable spell checking for python raw strings (or other language-specific contexts)

### ISSUE THREAD

**Issue by Jakob-Unfried** [streetsidesoftware/vscode-spell-checker#3016]
### Context
The spellchecker regularly gives me false positives when i use regex expressions in python.
E.g. `re.compile(r'#\s[Cc]opyright (20[0-9]{2}-)?20[0-9]{2}')` gives a false positive on `opyright`.

### Requested Feature
A user-friendly way to disable spell checking for anything inside a python raw-string, but keeping non-raw strings.
I would like to get an error on `print('Dont forget the opyright notice!')` but not on `re.compile(r'#\s[Cc]opyright (20[0-9]{2}-)?20[0-9]{2}')`.

This can probably be achieved by using the `cSpell.languageSettings` -> `ignoreRegExpList` setting, but i dont speak enough regex to figure out how to do it.
My particular problem could be solved by giving me the right regex for python raw strings, but it might be a nice 
~feature to have a  either a pre-defined set of language specific contexts, or just to give or link a list of useful regex in https://cspell.org/configuration/language-settings/ for the next time someone wants to achieve this for some other language-specific context~

EDIT: having dug up a bit more of the docs let me rephrase:
It would be nice to include raw strings in https://cspell.org/types/cspell-types/types/PredefinedPatterns.html

**Comment by Jason3S** [comment]
@Jakob-Unfried,

It is not easy to discover, but the Python dictionary already difines a pattern for raw string:
https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/python/cspell-ext.json#L37-L38

```jsonc
{
  "name": "string_raw",
  "pattern": "/\\bu?r(['\"]).*?\\1/gi"
},
```

**Comment by Jakob-Unfried** [comment]
Works just as intended, thanks!
